index.html

```diff
@@ -204,6 +204,18 @@ <h3 class="sub1">Multi-Post Stories</h3>
          </div>
       </div>
    </section>
+   <section class="contact">
+      <h2 class="contact-head">CONTACT US</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+      <div class="form">
+         <form method="post" action="https://formspree.io/f/mknyrjkd" class="form">
+            <input type="text" name="name" id="name" placeholder="Text Input" required maxlength="30" class="input">
+            <input type="email" name="email" required class="input" value="jeremy@gmail.com">

```

Comment:
- [ ] You are supposed to use placeholder texts for the input fields, Instead of the value attribute please use the placeholder attribute in your email input field.

Reply:
All good now 👍🏽